shell/browser/api/electron_api_protocol.cc

```diff
@@ -173,6 +178,11 @@ ProtocolError Protocol::RegisterProtocol(ProtocolType type,
                                          const std::string& scheme,
                                          const ProtocolHandler& handler) {
   bool added = protocol_registry_->RegisterProtocol(type, scheme, handler);
+  if (added) {
+    if (type == ProtocolType::kHttp || type == ProtocolType::kStream) {
+      g_streaming_schemes.push_back(scheme);
```

Comment:
Necessary review, this call assumes that `registerSchemeAsPrivileged` will be called after , but we document users to make the 'registerSchemeAsPrivileged` as the first call in the app as early as possible before app ready due to threading constraints in `//url`.

The right way would be to provide a new option `stream: true` in the `registerSchemeAsPrivileged` api and go from there.

Reply:
My assumption was that we should automatically register known-streaming protocols, but I guess you're saying that's not an option due to when `registerProtocol()` is called?

Reply:
> I guess you're saying that's not an option due to when registerProtocol() is called?

Yup